lib/system/memalloc.nim

```diff
@@ -33,6 +33,19 @@ when notJSnotNims:
     ## otherwise. Like any procedure dealing with raw memory this is
     ## **unsafe**.
 
+  proc cmpMem*(a, b: pointer, size: Natural): int {.inline, noSideEffect,
+    tags: [], locks: 0, raises: [].}
+    ## Compares the memory blocks ``a`` and ``b``. ``size`` bytes will
```

Comment:
```suggestion
    ## Compares the memory blocks `a` and `b`. `size` bytes will
```

single backticks are preferred. No need to change `equalMem` I think. Just don't introduce new ones.